content/applications/finance/documents/actions.rst

```diff
@@ -0,0 +1,117 @@
+
+#. **Owner**: the files must be associated with the owner set here.
+
+.. image:: actions/workflow-action-basic-conditions.png
+   :align: center
+   :alt: Example of a workflow action's basic condition in Odoo Documents
+
+.. tip::
+   If you don't set any condition, the action button appears for all files located inside the
+   workspace you selected.
+
+Advanced condition type: domain
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important:: It is recommended to have some knowledge of Odoo development to properly configure
+   *Domain* filters.
+
+To access the *Domain* condition, the :doc:`developer mode <../../general/developer_mode/activate>`
+needs to be activated. Once that's done, select the *Domain* condition type, and click on *Add
+Filter*.
+
+.. image:: actions/activate-domain-condition-type.png
+   :align: center
+   :alt: Activating the domain condition type in Odoo Documents
+
+To create a rule, you typically select a field, an operator, and a value. For example, if you want
+to add a workflow action to all the PDF files inside a workspace, set the field to: *Mime Type*, the
+operator to *contains*, and the value to *pdf*.
+
+.. image:: actions/domain-filter-example.png
+   :align: center
+   :alt: Example of a workflow action's domain condition in Odoo Documents
+
+Click on *Add node* (plus circle icon) and *Add branch* (ellipsis icon) to add conditions and
+sub-conditions. You can then specify if your rule should match *ALL* or *ANY* conditions. You can
+also edit the rule directly using the *Code editor*.
+
+.. image:: actions/add-branch-node-domain-condition.png
+   :align: center
+   :alt: Add a node or a branch to a workflow action's condition in Odoo Documents
+
+Configure the actions
+---------------------
+
+Select the *Actions* tab to set up your action. You can simultaneously:
+
+#. **Set Contact**: add a contact to the file, or replace an existing contact with a new one.
+
+#. **Set Owner**: add an owner to the file, or replace an existing owner with a new one.
+
+#. **Move to Workspace**: move the file to any workspace.
+
+#. **Create**: create one of the following items attached to the file in your database:
+
+   1. **Product template** (create a product you can edit directly)
+   2. **Task** (create a Project task you can edit directly)
+   3. **Signature request** (create a new Sign template to send out)
+   4. **Sign directly** (create a Sign template to sign directly)
+   5. **Vendor bill** (create a vendor bill using OCR and AI to scrape information from the file
+      content)
+   6. **Customer invoice** (create a customer invoice using OCR and AI to scrape information from
+      the file)
+   7. **Vendor credit note** (create a vendor credit note using OCR and AI to scrape information
+      from the file)
+   8. **Credit note** (create a customer credit note using OCR and AI to scrape information from the
+      file)
+   9. **Applicant** (create a new HR application you can edit directly)
```

Comment:
not sure why you put the description between brackets rather than presenting it like above?
```suggestion
  - **Product template**: create a product you can edit directly
  - **Task**: create a Project task you can edit directly
  - **Signature request**: create a new Sign template to send out
  - **Sign directly** (create a Sign template to sign directly)
  - **Vendor bill** (create a vendor bill using OCR and AI to scrape information from the file
      content)
  - **Customer invoice** (create a customer invoice using OCR and AI to scrape information from
      the file)
  - **Vendor credit note** (create a vendor credit note using OCR and AI to scrape information
      from the file)
  - **Credit note** (create a customer credit note using OCR and AI to scrape information from the
      file)
  - **Applicant** (create a new HR application you can edit directly)
```